extension/src/editor/containers/ui-components/property/PropertyOperationSelectField.tsx

```diff
@@ -0,0 +1,45 @@
+import { useCallback } from "react";
+
+import { PropertySelectField } from "./PropertySelectField";
+
+const operationOptions = [
+  { value: ">", label: ">" },
+  { value: "<", label: "<" },
+  { value: ">=", label: ">=" },
+  { value: "<=", label: "<=" },
+  { value: "===", label: "=" },
+  { value: "!==", label: "!=" },
+] as const;
+
+export type OperationValue = (typeof operationOptions)[number]["value"] | undefined;
```

Comment:
I made new component to handle operations. And I fixed to use `OperationValue` type for each operation of condition like [here](https://github.com/eukarya-inc/PLATEAU-VIEW-3.0/pull/72/files#diff-818ccc944200f9844185bf191ea1c60f1132f5d04e965aa5ba955e202095542dR28).